src/components/Catalog.vue

```diff
@@ -397,15 +404,17 @@ export default {
           } else {
             this.externalItemCount = items.features.length;
           }
+          ['first', 'previous', 'next', 'last'].forEach(linkRelation => {
+            const link = items.links.find(link => link.rel === linkRelation);
```

Comment:
To ensure href is available

```suggestion
            const link = items.links.find(link => link && link.rel === linkRelation && typeof link.href === 'string');
```

Reply:
👍 I also saw that I could switch to
 ```javascript
Object.keys(this.externalPaginationLinks).forEach(linkRelation => {
```
Instead of the hardcoded array, so that we only need to change `this.externalPaginationLinks` structure (children) and everything else will follow. Sounds good?

Reply:
Yes, that is good!